03.bowling/frame.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+class Frame
+  attr_reader :shots
+
+  def initialize(shots, is_last_frame: false)
+    @shots = shots
+    @is_last_frame = is_last_frame
+  end
+
+  def score(index, all_frames)
+    base_score = @shots.sum(&:pins)
+    base_score = adjust_score_for_last_frame(base_score)
```

Comment:
```suggestion
    base_score = @is_last_frame ? calc_base_score_for_last : calc_base_score
```

みたいにして再代入を避けてあげるといいかも、と思いました。
https://bootcamp.fjord.jp/pages/avoid-re-assign